### ISSUE TITLE
Rename to Starlib or something

### ISSUE THREAD

**Issue by arturdryomov** [bazelbuild/bazel-skylib#218]
* [Skylark → Starlark](https://blog.bazel.build/2018/08/17/starlark.html)
* Skydoc → [Stardoc](https://github.com/bazelbuild/stardoc)
* Skylib → Starlib?

**Comment by brandjon** [comment]
I agree that "Starlib" is a more proper name for this library. However it might be a breaking change -- certainly for WORKSPACE logic, and perhaps also for other reasons. I wouldn't want to force such a change on users unless it's tied to other breaking changes that we deem necessary, e.g. a total API redesign (which is unlikely). Since we aren't committing to a direction for the API either way at the moment, it doesn't make sense to rename now.
